src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -410,6 +468,21 @@
      */
     public void setAuthenticationScheme(String authenticationScheme);
 
+    /**
+     * sets the authentication mode
+     * 
+     * @param authentication
+     *            the authentication mode
+     */
+    public void setAuthentication(String authentication);
```

Comment:
similar to above, why isn't this setAuthenticationMode(String authenticationMode)?